external/microprofile/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_library(microprofile STATIC
+    microprofile/microprofile.h
+    microprofile/microprofile_html.h
+    microprofile/microprofile.cpp
+)
+
+target_include_directories(microprofile
```

Comment:
I just tried to compile this branch and got this error:
```
/usr/bin/ld: ../external/microprofile/libmicroprofile.a(microprofile.cpp.o): undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
//usr/lib64/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```
Adding
```
find_package(Threads)
target_link_libraries(microprofile
    PUBLIC
        Threads::Threads
)
```
fixed this.

Reply:
This is still required.
This link error does not show up on ci because `ENABLE_PROFILERS=address` on ci.
If I enable this profiler, the link error goes away.

The link error happens for rwviewer, since that one does not link to SDL2 (which already links to pthread).